.github/ci/spack-envs/clang7_nopy_ompi_h5_ad2/spack.yaml

```diff
@@ -12,7 +12,7 @@ spack:
 
   packages:
     adios2:
-      variants: ~zfp ~sz ~png ~dataman ~python ~fortran ~ssc ~shared ~bzip2
+      variants: ~zfp ~sz ~png ~dataman ~python ~fortran ~ssc ~shared ~bzip2 ~blosc2
```

Comment:
That we need to disable blosc2 is concerning, but maybe we can disable it again with #1597